dev-docs/bidders/minutemedia.md

```diff
@@ -4,7 +4,7 @@ title: MinuteMedia
 description: Prebid Bidder Adaptor
 pbjs: true
 biddercode: minutemedia
-media_types: video
+media_types: banner, video
```

Comment:
Thanks for updating the docs. Could you add the flag `multiformat_supported` (introduced in #3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.

Reply:
done